ledger/src/bank_forks_utils.rs

```diff
@@ -115,6 +122,10 @@ pub fn load_bank_forks(
         false
     };
 
+    let exit = Arc::new(AtomicBool::new(false));
+    let replayer = Replayer::new(get_thread_count(), &exit);
+    let replayer_handle = replayer.handle();
```

Comment:
i don't think these threads ever finish since we don't call join or set exit to true.

Reply:
yup, need to clean that up